.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,40 @@
+# template from https://github.com/JamesIves/github-pages-deploy-action/tree/dev
+
+name: CI
+
+run-name: Deployment Triggered - ${{ github.event.workflow_run.conclusion == 'success'}}
+
+on:
+  workflow_run:
+    workflows: ["Run tests"]
+    types:
+      - completed
```

Comment:
This is really nice, did you test that it only starts this workflow when the test work flow has succeeded?
There's just one thing to add, though, we should only publish on push to `main`, there should be an option you can add under `workflow_run` to make sure it only happens on a push to main, and not on a PR or on other branches.

Reply:
I should have described it but yes I tested it! In my test fork I basically replaced the tests with "run: exit 1" and the way it's set up, it will always look like this workflow ran, but it's because it has to run to check if the previous tests fail. If it fails, it runs the other job which essentially nothing.

I was wondering about that because I was unsure about when the tests should be run. I'll change it to only work for main.